salt/etcd/create-or-update-etcd-pillar

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+docker exec $(docker ps -a | grep velum-dashboard | awk '{print $1}') entrypoint.sh bundle exec rails runner 'ActiveRecord::Base.logger=nil; pillar = Pillar.find_or_create_by(pillar: "api:etcd_version"); pillar.value = "etcd3"; pillar.save'
```

Comment:
Oh, `docker ps -a` may catch a "dead" container, we need the live/current one, so dropping the "-a" will make this more reliable.

Reply:
You are right, fixed